README.md

```diff
@@ -84,6 +84,7 @@ ATIS Kutaisi 251.000, TRAFFIC 252.000, VOICE GC:en-US-Wavenet-B
 ATIS Kutaisi 251.000, TRAFFIC 252.000, VOICE AWS:Nicole
 ATIS Kutaisi 251.000, TRAFFIC 252.000, VOICE WIN
 ATIS Kutaisi 251.000, TRAFFIC 252.000, INFO Q, ACTIVE 21L, NO QFE, NO HPA
+ATIS Kutaisi 251.000, TRAFFIC 252.000, INFO Q, ACTIVE 21L.03R, NO QFE, NO HPA
```

Comment:
Thanks for the PR! Just a nit, what do you think about using a `/` instead of a `.` to separate the arrival and departure runways? Would seem easier to understand to me, but maybe the `.` is a common denotation for that I don't know?

Reply:
I used a dot because it fulfilled the existing grammar rules that the message was checked for.
I can make his work with /, too, but will have to investigate.

Reply:
Gotcha, I'll make the change. Thanks for the PR again 👍